src/client/voice/dispatcher/StreamDispatcher.js

```diff
@@ -142,6 +142,15 @@ class StreamDispatcher extends Writable {
     if (this._writeCallback) this._writeCallback();
   }
 
+  /**
+   * Stops the current stream permanently and emits a `finish` event.
+   * @param {string} reason The reason for ending this stream.
```

Comment:
Shouldn't this reason be wrapped in brackets as its's optional?

Reply:
Good spot!